tests/testthat/test-set_analysis_inputs_path.R

```diff
@@ -0,0 +1,57 @@
+test_that("sets appropriate global variables", {
+  data_location_ext <- "xxx"
+
+  output <-
+    callr::r(
+      func = function(set_analysis_inputs_path, data_location_ext) {
+        set_analysis_inputs_path(data_location_ext)
+      },
+      args = list(set_analysis_inputs_path, data_location_ext)
+    )
+
+  expect_equal(output, data_location_ext)
+})
+
+test_that("sets appropriate global variables", {
+  data_location_ext <- "xxx"
+  proj_data_location_ext <- "yyy"
+
+  output <-
+    callr::r(
+      func = function(set_analysis_inputs_path,
+                      data_location_ext,
+                      proj_data_location_ext) {
+        .GlobalEnv$proj_data_location_ext <- proj_data_location_ext
+        set_analysis_inputs_path(data_location_ext)
+      },
+      args = list(
+        set_analysis_inputs_path,
+        data_location_ext,
+        proj_data_location_ext
+      )
+    )
+
+  expect_equal(output, proj_data_location_ext)
+})
+
+test_that("sets appropriate global variables", {
```

Comment:
same as above, what behaviour is this testing for that the previous test isn't? it's pretty difficult for me to understand what is going on here

Reply:
```suggestion
test_that("returns expected value when overridden by a global variable", {
```
These are wrong and were carried over from another test. Is this better?

Reply:
The point is that there are two different tests with the exact same name, so it's pretty difficult to understand what the difference is that they are testing?

The suggestion you gave is still the same for both which would still be confusing